src/ontology/mondo-edit.obo

```diff
@@ -105869,20 +105869,22 @@ property_value: RO:0002175 NCBITaxon:9940 {source="https://orcid.org/0000-0002-4
 id: MONDO:0005676
 name: borna disease
 def: "An encephalomyelitis of horses, sheep and cattle caused by borna disease virus." [MESH:D001890]
-subset: nord_rare {source="MONDO:NORD"}
-subset: otar {source="MONDO:OTAR"}
-subset: rare
-synonym: "enzootic encephalomyelitis" EXACT [DOID:5154]
-xref: DOID:5154 {source="MONDO:equivalentTo", source="EFO:0007178"}
-xref: EFO:0007178 {source="MONDO:equivalentTo", source="MONDO:EFO"}
-xref: MESH:D001890 {source="MONDO:equivalentTo", source="DOID:5154", source="EFO:0007178"}
+comment: In Mondo we make the distinction between human and non-human animal diseases. Cross-references do not necessarily make this distinction. Annotate to the human or non-human animal term as appropriate. This disease is considered zoonotic. {source="PMID:31924550", source="PMID:34376142"}
```

Comment:
- 'comment' should not have a source. Instead, please add the PMID in the text.
- Here is an alternate text:
This disease is considered zoonotic, i.e. able to be transmitted between human and non-human animals (PMID:34376142). In Mondo, we distinguish between human and non-human animal diseases, and we currently do have a general term to represent a species-agnostic version of the disease. This record is for the non-human version of “borna disease”.

Reply:
Revised comment: This disease is considered zoonotic, i.e. able to be transmitted between human and non-human animals (PMID:34376142). In Mondo, we distinguish between human and non-human animal diseases, and we currently do not have a general term to represent a species-agnostic version of the disease. This record is for the non-human version of “borna disease”.